web/src/todo_widget.ts

```diff
@@ -440,33 +447,46 @@ export function activate({
 
         $elem.find("button.add-task").on("click", (e) => {
             e.stopPropagation();
-            $elem.find(".widget-error").text("");
+
             const task = $elem.find<HTMLInputElement>("input.add-task").val()?.trim() ?? "";
             const desc = $elem.find<HTMLInputElement>("input.add-desc").val()?.trim() ?? "";
 
-            if (task === "") {
-                return;
-            }
-
             $elem.find("input.add-task").val("").trigger("focus");
             $elem.find("input.add-desc").val("");
 
-            const task_exists = task_data.name_in_use(task);
-            if (task_exists) {
-                $elem.find(".widget-error").text($t({defaultMessage: "Task already exists"}));
-                return;
-            }
-
             const data = task_data.handle.new_task.outbound(task, desc);
             callback(data);
         });
     }
 
+    function update_add_task_button(): void {
+        const task = $elem.find<HTMLInputElement>("input.add-task").val()?.trim() ?? "";
+        const task_exists = task_data.name_in_use(task);
+        const $add_task_wrapper = $elem.find(".add-task-wrapper");
+        const $add_task_button = $elem.find("button.add-task");
+
+        if (task === "") {
+            $add_task_wrapper.attr(
+                "data-tippy-content",
+                $t({defaultMessage: "Name the task before adding."}),
```

Comment:
Is this really what the issue says? Probably we should do a different string; can you post a screenshot of this on the chat.zulip.org thread for this issue for feedback?